src/vs/workbench/browser/parts/editor/editorDropTarget.ts

```diff
@@ -501,6 +502,10 @@ class DropOverlay extends Themable {
 
 	private getOverlayOffsetHeight(): number {
 		if (!this.groupView.isEmpty && this.accessor.partOptions.showTabs) {
+			const tablist = document.querySelector(DropOverlay.TABS_SELECTOR) as HTMLElement;
```

Comment:
This will not work properly once you have more than one editor group opened because then you will have more than one `div.tabs-container`. 

I suggest an alternate solution where the `accessor` provides a method to ask for the height of the editor title area. The `accessor` in this case is the `EditorPart` which can delegate this to the group in question and then the group can forward this to the title control. And I would only use the `offsetHeight` property if mutli-line tabs are enabled.

Reply:
Done, please let me know if I've done this incorrectly.